Gruntfile.js

```diff
@@ -116,10 +115,9 @@ module.exports = function(grunt) {
   grunt.registerTask('test', [
     'dist',
     'eslint',
-    'exec:test',
+    'fork',
```

Comment:
https://github.com/mpc-hc/mpc-hc.org/blob/master/Gruntfile.js#L282

I guess `exec-test` be it then...